test/run.d

```diff
@@ -145,7 +181,29 @@ auto predefinedTargets(string[] targets)
         return testPath(dir).dirEntries("*{.d,.sh}", SpanMode.shallow).map!(e => e.name);
     }
 
-    Appender!(string[]) newTargets;
+    static Target createUnitTestTarget()
+    {
+        Target target = {
+            args: [getEnvironment["DMD"], "-run", "unit_test_runner.d"]
+        };
+
+        return target;
+    }
+
+    static Target createDefaultTarget(string filename)
+    {
+        Target target = {
+            filename: filename,
+            args: [
+                resultsDir.buildPath("d_do_test"),
+                Target.normalizedTestName(filename)
```

Comment:
This wasn't required before, why is it required now?

Reply:
Because the targets are not normalized at the beginning as before:

https://github.com/dlang/dmd/pull/8528/files#diff-060ed063c39ccc738bd0e384c097973cL70